druid/src/command.rs

```diff
@@ -301,7 +301,7 @@ impl std::fmt::Display for ArgumentError {
             ArgumentError::WrongVariant => write!(
                 f,
                 "Incorrect access method for argument type; \
-            check Command::one_shot docs for more detail."
+                 check Command::one_shot docs for more detail."
```

Comment:
I approve of your (or your IDE's) attention to detail,

Reply:
I thought your CI ran `cargo fmt --check` so i always run `cargo fmt` before a MR
Would you like me to look at adding it to the CI?

Reply:
it does run, but I didn't think `cargo fmt` cares about whitespace in this particular scenario?